AEPMessagingLiveActivity/Sources/LiveActivityData.swift

```diff
@@ -19,27 +19,43 @@ import Foundation
 /// `LiveActivityAttributes` protocol.
 @available(iOS 16.1, *)
 public struct LiveActivityData: Codable {
-    /// Unique identifier for managing and tracking an individual Live Activity in Adobe Experience Platform.
-    public var liveActivityID: String?
+    /// Indicates the source of the Live Activity's creation.
+    ///
+    /// Use this enum to distinguish whether a Live Activity was started locally by the app
+    /// or remotely via push-to-start delivered by the server.
+    public enum Origin: String, Codable {
+        /// The Live Activity was initiated locally on the device.
+        case local
+        /// The Live Activity was initiated remotely via a push-to-start token.
+        case remote
+    }
 
     /// Unique identifier for managing and tracking a broadcast Live Activity channel in Adobe Experience Platform.
-    public var channelID: String?
+    public let channelID: String?
+
+    /// Unique identifier for managing and tracking an individual Live Activity in Adobe Experience Platform.
+    public let liveActivityID: String?
+
+    /// Defines whether the Live Activity was started locally by the app or remotely via a push-to-start notification.
+    public let origin: Origin
 
-    /// Creates an `LiveActivityData` instance with the specified Live Activity ID.
-    /// Use this method for Live Activities for an individual person.
+    /// Initializes a `LiveActivityData` instance with the specified broadcast channel ID.
+    /// Use this initializer for Live Activities broadcast to subscribers of a channel.
     ///
-    /// - Parameter liveActivityID: The unique identifier for the Live Activity.
-    /// - Returns: A new `LiveActivityData` instance populated with the provided `liveActivityID`.
-    public static func create(liveActivityID: String) -> LiveActivityData {
-        LiveActivityData(liveActivityID: liveActivityID)
+    /// - Parameter channelID: The unique identifier for the Live Activity broadcast channel.
+    public init(channelID: String) {
```

Comment:
thoughts on condensing the initializers into one and providing some extra flexibility?
```
public init(channelID: String? = nil, liveActivityID: String? = nil, origin: LiveActivityOrigin = .local)
```
covers every path...